src/wp-includes/option.php

```diff
@@ -2037,6 +2037,25 @@ function get_network_option( $network_id, $option, $default_value = false ) {
 	 */
 	$pre = apply_filters( "pre_site_option_{$option}", false, $option, $network_id, $default_value );
 
+	/**
+	 * Filters the value of an existing network options before it is retrieved.
+	 *
+	 * Returning a truthy value from the filter will effectively short-circuit retrieval
+	 * and return the passed value instead.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param mixed  $pre         The value to return instead of the network option value. This differs
+	 *                            from `$default`, which is used as the fallback value in the event
+	 *                            the option doesn't exist elsewhere in get_network_option().
+	 *                            Default false (to skip past the short-circuit).
```

Comment:
```suggestion
	 *                            Default false (to skip past the short-circuit). If this is not false, it was changed by `pre_site_option_{$option}`
```

I think there is value on being clear about when this wouldn't be false.